spec/features/associating_topics_to_mainstream_browse_pages_spec.rb

```diff
@@ -6,10 +6,28 @@
     stub_all_panopticon_tag_calls
   end
 
-  context "existing mainstream browse pages" do
+  context "parent mainstream browse pages" do
     let!(:mainstream_browse_page) { create(:mainstream_browse_page) }
-    let!(:topic)                  { create(:topic) }
-    let!(:topic_two)              { create(:topic) }
+
+    it "should not allow adding any topics" do
+      visit edit_mainstream_browse_page_path(mainstream_browse_page)
+
+      within "form.resource" do
+        expect(page).to_not have_selector("#mainstream_browse_page_topics")
+      end
+    end
+  end
+
+  context "existing mainstream browse pages" do
+    let!(:mainstream_browse_page_parent) { create(:mainstream_browse_page) }
+    let!(:mainstream_browse_page)        { create(:mainstream_browse_page) }
```

Comment:
You could change this to be `create(:mainstream_browse_page, :parent => mainstream_browse_page_parent)`, and then you wouldn't need the before block below...